public/assets/scss/_index.scss

```diff
@@ -281,3 +281,228 @@ h5 .btn-link {
 .break {
     display: inline-block;
 }
+
+/* big tablets */
+@media only screen and (max-width: 1024px) {
```

Comment:
My opinion on ensuring that the responsive text are added inline still holds as explained here - https://github.com/Highrachy/ballers-ws/pull/12#discussion_r408684581. However, I am comfortable with you fixing it in another PR.